javascript/exorelay/features/support/steps-given.ls

```diff
@@ -19,6 +20,12 @@ module.exports = ->
     @exo-relay = new ExoRelay {exocom-host: 'localhost', @exocom-port, role: @role = 'test-service'}
 
 
+  @Given /^an ExoRelay instance is connected at port (\d+)$/ (@exocom-port, done) ->
+    @exo-relay = new ExoRelay {exocom-host: 'localhost', @exocom-port, role: @role = 'test-service'}
+      ..connect!
+      ..on 'online', ~> done!
```

Comment:
This looks weird, but apparently ExoRelay makes a callback with the Exocom port as the first parameter of an `'online'` event. We're not using it so I figured I'd ignore it. We should probably make a separate PR that changes that callback to `(err, port)`